.circleci/config.yml

```
@@ -603,78 +359,196 @@ workflows:
+          postgres: 11
+          tag: 11.10-dev202012151914
+          make: check-worker
           requires: [build-11]
-      - test-11_check-follower-cluster:
+      - test-citus:
+          name: 'test-11_check-follower-cluster'
+          postgres: 11
+          tag: 11.10-dev202012151914
+          make: check-follower-cluster
           requires: [build-11]
-      - test-11_check-failure:
+      - test-citus:
+          name: 'test-11_check-failure'
+          postgres: 11
+          image: citus/failtester
+          tag: 11.10-dev202012151914
+          make: check-failure
           requires: [build-11]
 
-      - test-12_check-multi:
+      - test-citus:
+          name: 'test-12_check-multi'
+          postgres: 12
+          tag: 12.5-dev202012151914
+          make: check-multi
           requires: [build-12]
-      - test-12_check-vanilla:
+      - test-citus:
+          name: 'test-12_check-multi-mx'
+          postgres: 12
+          tag: 12.5-dev202012151914
+          make: check-multi-mx
           requires: [build-12]
-      - test-12_check-isolation:
+      - test-citus:
+          name: 'test-12_check-vanilla'
+          postgres: 12
+          tag: 12.5-dev202012151914
+          make: check-vanilla
           requires: [build-12]
-      - test-12_check-mx:
+      - test-citus:
+          name: 'test-12_check-isolation'
+          postgres: 12
+          tag: 12.5-dev202012151914
+          make: check-isolation
           requires: [build-12]
-      - test-12_check-worker:
+      - test-citus:
+          name: 'test-12_check-worker'
+          postgres: 12
+          tag: 12.5-dev202012151914
+          make: check-worker
           requires: [build-12]
-      - test-12_check-follower-cluster:
+      - test-citus:
+          name: 'test-12_check-follower-cluster'
+          postgres: 12
+          tag: 12.5-dev202012151914
+          make: check-follower-cluster
           requires: [build-12]
-      - test-12_check-columnar:
+      - test-citus:
+          name: 'test-12_check-columnar'
+          postgres: 12
+          tag: 12.5-dev202012151914
+          make: check-columnar
           requires: [build-12]
-      - test-12_check-columnar-isolation:
+      - test-citus:
+          name: 'test-12_check-columnar-isolation'
+          postgres: 12
+          tag: 12.5-dev202012151914
+          make: check-columnar-isolation
           requires: [build-12]
-      - test-12_check-failure:
+      - test-citus:
+          name: 'test-12_check-failure'
+          postgres: 12
+          image: citus/failtester
+          tag: 12.5-dev202012151914
+          make: check-failure
           requires: [build-12]
 
-      - test-13_check-multi:
+      - test-citus:
+          name: 'test-13_check-multi'
+          postgres: 13
+          tag: 13.1-dev202012151914
+          make: check-multi
           requires: [build-13]
-      - test-13_check-vanilla:
+      - test-citus:
+          name: 'test-13_check-multi-mx'
```

revert back to old name of test. that will prevent us from having to change github settings

```suggestion
          name: 'test-13_check-mx'
```